README.md

```diff
@@ -351,6 +351,28 @@ This will run the local development web server and make the admin site
 available at http://localhost:6012; remember to make sure that the Notify.gov
 API is running as well!
 
+## Creating a 'First User' in the database
+
+After you have completed all setup steps, you will be unable to log in, because there
+will not be a user in the database to link to the login.gov account you are using.  So
+you will need to create that user in your database using the 'create-test-user' command.
+
+Open two terminals pointing to the api project and then run these commands in the
+respective terminals.  Make sure the email address is the same one you are using in
+login.gov and make sure your phone number is in the format 5555555555.
+
+(Server 1)
+env ALLOW_EXPIRED_API_TOKEN=1 make run-flask
+
+(Server 2)
+poetry run flask command create-admin-jwt | tail -n 1 | pbcopy
+poetry run flask command create-test-user --name="<Your Name>" --email="<email address>" --mobile_number="<your phone>" --password="<password>" --admin=True;
```

Comment:
Is it wise to have a line like this with a password exposed in command-line history for the person who runs it? Seems better to allow the prompts to run.... I don't think it is necessarily a problem as this is a local instance, but just screams "antipattern" to me, since the command history for 99.9% of users is unencrypted, and the password would be in plain text for anyone who gains access to that command-line history to see.

Reply:
If someone has access to my local machine, this is probably not going to be their highest priority target.  But I will change it.

Reply:
I mean, true. But still, best practices ;)